instruqt-tracks/sentinel-cli-basics/track.yml

```diff
@@ -1,9 +1,4 @@
-# Copyright (c) HashiCorp, Inc.
```

Comment:
instruqt likes to strip out our copyright header, we'll need to add this back or license-bot will complain.

Reply:
@thomashashi i created another commit for adding the header back.